db/import_listener.go

```diff
@@ -28,6 +28,7 @@ type importListener struct {
 	stats            *base.DatabaseStats // Database stats group
 	cbgtContext      *base.CbgtContext   // Handle to cbgt manager,cfg
 	checkpointPrefix string              // DCP checkpoint key prefix
+	loggingCtx       context.Context     // ctx for logging, since function signatures without ctx are required by external dependency, for ex.: sgbucket.FeedEventCallbackFunc
```

Comment:
sgbucket is owned by us, so isn't an external dependency and could be changed.  If there are other cases (like cbgt), maybe mention those here instead.

Reply:
oh ok, not sure how/if cbgt is used here yet, but I'll note this is for event callbacks, thanks.